Simplify docker compose usage

tnonnenm opened the pull request:
Create a docker compose file to simply start application together with DB as compose.

Add hint howto get the GIT tag into the local build docker compose

StevenWeathers commented:
Not sure the real value in this, the docker-compose file would still need to be modified as all the passwords/secrets would not be ideal to utilize outside of local development.  As far as updating the docker-compose directions to include version tag doesn't really feel too useful, for local development its pretty unnecessary to display the app version in the ui.

tnonnenm commented:
I added the docker-compose file because when I was trying to get this to work back then I had some problems with the build and I was not so familiar with docker at all. This is just for people that have no idea how this works (like me a few months ago) and want to have a docker-compose file that set's up everything for them...

The second part was because I added a REST interface to the application (see PR #124) that is not yet in the master ... It was ugly with a lot of hard coded stuff that would only work in our environment. So I created an image on my machine and transferred it to our server. This is why I added the comments to the Readme.

This was my purpose ;-)

StevenWeathers commented:
> I added the docker-compose file because when I was trying to get this to work back then I had some problems with the build and I was not so familiar with docker at all. This is just for people that have no idea how this works (like me a few months ago) and want to have a docker-compose file that set's up everything for them...
> 
> The second part was because I added a REST interface to the application ... It started as dirty hack with a lot of hard coded stuff that would only work in our environment. I had to build an image on my machine and transfer it to our server. This is why I added the comments to the Readme.
> In the past two days I managed to bring the feature to a pull request (see PR #124)
> 
> This was my purpose ;-)

Thanks I understand the challenges of learning new tools like Docker, however since the docker-compose file adds no real value being in the main repo I'm going to have to decline this one.  I'm all for updating the readme with any helpful information, maybe links to docker's documentation on how to use docker-compose.
